package.json

```diff
@@ -51,7 +51,6 @@
     "react-bootstrap": "^0.30.6",
     "react-dom": "^15.3.2",
     "react-input-enhancements": "^0.5.3",
-    "react-pure-render": "^1.0.2",
```

Comment:
I removed it because it was a duplicate package, it is still required in the `dependencies` section.